### PR TITLE
docs(bridge): Add documentation and settings for IntelliJ and VSCode(#4739)

### LINKED ISSUE #4739: Document and provide easy IDE setup for Bridge devs
For new Bridge developers and also for contributers it should be easy to have a IDE setup that works with our contribution guidelines (e.g. linting). We could also provide a settings file for common IDEs like IntelliJ and VSCode.

Therefore I suggest to add a section in the Bridge README, to make onboarding new developers easy and have the same setup for all developers.

# Definition of Done

- [x] Documentation about IDE setup in README file including documentation about Linting settings (TSLint, SonarLint, ...)
- [x] Settings files downloadable from repository for IntelliJ and VSCode

### PR DESCRIPTION
**Pull request by laneli**
## This PR
Closes #4739
Adds documentation and setting files for IntelliJ and Visual Studio code on how to set up the IDE.

### CODE REVIEW

File: bridge/README.md [hunk 1]
```diff
@@ -120,10 +132,118 @@ kubectl delete -f deploy/service.yaml
...
-1. Run `npm run start:dev` from bridge root level to start the express server and the Angular app.
-1. Access the web through the url shown on the console (e.g., http://localhost:3000/ ).
+4. Run `npm run start:dev` from bridge root level to start the express server and the Angular app.
+5. Access the web through the url shown on the console (e.g., http://localhost:3000/ ).
+
+## IDE Setup
+
+### Create workspace
+
+Before creating the workspace, make sure that you already cloned the keptn/keptn repository to your local file system.<br/>
```
Comment (ermin-muratovic) [inline bridge/README.md:142]: The `<br/>` is not necessary here I guess?

**Review by ermin-muratovic — CHANGES_REQUESTED** [review]

File: bridge/README.md (hunk 1 above)
Comment (laneli) [reply bridge/README.md:142]: I would leave it as it is - as it is additional information for IntelliJ and is then more readable / visible :)

Comment (laneli) [reply bridge/README.md]: Removed it 😆

**Review by ermin-muratovic — APPROVED** [review]
lgtm

**Pull request merged**